core/lib/dependency-graph/simulator/tcp-connection.js

```diff
@@ -176,12 +178,34 @@ class TcpConnection {
     const extraBytesDownloaded = this._h2 ? Math.max(totalBytesDownloaded - bytesToDownload, 0) : 0;
     const bytesDownloaded = Math.max(Math.min(totalBytesDownloaded, bytesToDownload), 0);
 
+    /** @type {ConnectionTiming} */
+    let connectionTiming;
+    if (!this._warmed) {
+      connectionTiming = {
+        dnsResolutionTime,
+        connectionTime: handshakeAndRequest - dnsResolutionTime,
+        sslTime: this._ssl ? twoWayLatency : undefined,
+        timeToFirstByte,
+      };
+    } else if (this._h2) {
+      // TODO: timing information currently difficult to model for warm h2 connections.
+      connectionTiming = {
+        timeToFirstByte,
+      };
```

Comment:
the way warm h2 connections are simulated means there's not really anything meaningful going on here (`timeToFirstByte` will actually always be 0), but that's something I'd like to iterate on